kolibri/plugins/learn/assets/src/views/LearnIndex.vue

```diff
@@ -161,6 +171,15 @@
             immersivePageIcon: 'close',
           };
         }
+        if (this.pageName === PageNames.CUSTOM_TOPICS_CHANNEL) {
```

Comment:
Can reuse your computed prop above here (and then if we change the implementation to not be contingent on page name as I think we will need to, then you don't have to encode twice).